src/geometries/polytopes/pyramid.jl

```diff
@@ -6,11 +6,18 @@
     Pyramid(p1, p2, p3, p4, p5)
 
 A pyramid with points `p1`, `p2`, `p3`, `p4`, `p5`.
+The first four points correspond to the vertices of the base and the last is the top of the pyramid.
```

Comment:
```suggestion
The first four points correspond to the vertices of the base and the last point is the apex..
```
I think Júlio's formulation using `apex` was more accurate than my suggestion using `top` 😄

Reply:
Ok, you beat me on it :sweat_smile: